iam/api-client/src/main/java/iam/snippets/CreateServiceAccountKey.java

```diff
@@ -16,80 +16,36 @@
 package iam.snippets;
 
 // [START iam_create_key]
-import com.google.api.client.googleapis.javanet.GoogleNetHttpTransport;
-import com.google.api.client.json.gson.GsonFactory;
-import com.google.api.services.iam.v1.Iam;
-import com.google.api.services.iam.v1.IamScopes;
-import com.google.api.services.iam.v1.model.CreateServiceAccountKeyRequest;
-import com.google.api.services.iam.v1.model.ServiceAccountKey;
-import com.google.auth.http.HttpCredentialsAdapter;
-import com.google.auth.oauth2.GoogleCredentials;
+
+import com.google.cloud.iam.admin.v1.IAMClient;
+import com.google.iam.admin.v1.CreateServiceAccountKeyRequest;
+import com.google.iam.admin.v1.ServiceAccountKey;
 import java.io.IOException;
-import java.security.GeneralSecurityException;
-import java.util.Base64;
-import java.util.Collections;
 
 public class CreateServiceAccountKey {
 
```

Comment:
For all snippets, add a main method which inturn calls the actual method.
Add parameter descriptions as part of the main method.